libr/include/r_cons.h

```diff
@@ -447,8 +447,8 @@ typedef struct r_cons_context_t {
 	RConsGrep grep;
 	RStack *cons_stack;
 	char *buffer;
-	int buffer_len;
-	int buffer_sz;
+	ut64 buffer_len;
```

Comment:
What about size_t?

Reply:
Sure, could be size_t aswell probably, I think size_t is guaranteed to hold size of any object so it should cover capacity of whole possible memory.

Reply:
however size_t could be 32bit on 32bit system. Would this risk to reintroduce the wrong behaviour on those platforms?

Reply:
well 32bit systems wouldn't fit more 2^32 bytes of output anyways

Reply:
Good point :D :D

Reply:
use size_t and u get my +1